XmlFormatter/src/Manager/VersionManager.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Octokit;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using XmlFormatter.src.DataContainer;
+using XmlFormatter.src.EventMessages;
+
+namespace XmlFormatter.src.Manager
+{
+    class VersionManager
+    {
+        public event EventHandler<BaseEventArgs> Error;
+
+        public VersionManager()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=4995221)